regcore_read/views/es_search.py

```diff
@@ -37,6 +39,10 @@ def search(request, doc_type):
             term['version'] = version
         if regulation:
             term['regulation'] = regulation
+        if is_root:
+            term['is_root'] = is_root
+        if is_subpart:
+            term['is_subpart'] = is_subpart
```

Comment:
Weirdly, these actually need to *not* be converted to booleans. The haystack search wants to filter on `"true"` and `"false"`, not `True` and `False`. Passing a boolean to e.g. `filter(is_root=False)` yields unexpected results.